src/register.jl

```diff
@@ -624,6 +624,7 @@ function register(
     if !isempty(package_repo)
         package_repo = GitTools.normalize_url(package_repo)
     end
+    package_repo = replace(package_repo, "github" => "GitHub")
```

Comment:
```suggestion
    package_repo = replace(package_repo, r"^https?://github\.com/"i => "https://GitHub.com/")
```
Don't want to change "github" to "GitHub" elsewhere in the URL.